module/pom.xml

```diff
@@ -10,7 +10,7 @@
     <groupId>org.atmosphere</groupId>
     <artifactId>atmosphere-play</artifactId>
     <name>Atmosphere Play Module</name>
-    <version>2.4.1-SNAPSHOT</version>
+    <version>2.4.2</version>
```

Comment:
Do not change the version.It will be automatically bumped when the official release happens